Description: Relax dependency of this error
 umask package pin thiserror lib to 1.0.40 that is not packaged yet
 in Debian. Relaxing the thiserror lib version to 1.0.38
Author: Emmanuel Arias <eamanu@debian.org>
Last-Update: 2023-09-19 
Index: umask/Cargo.toml
===================================================================
--- umask.orig/Cargo.toml	2023-09-18 11:18:18.000000000 -0300
+++ umask/Cargo.toml	2023-09-19 11:17:43.385754600 -0300
@@ -27,4 +27,4 @@
 repository = "https://github.com/Canop/umask"
 
 [dependencies.thiserror]
-version = "1.0.40"
+version = "1.0.38"
